### DISCUSSION TITLE
Can we use Nextjs (react) instead ov VUE?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [shopware/frontends#479] [General]
Hi all, I'd like to setup a headless storefront with react, I can use the shopware client library right? No matter of which js framework I use.
Also wanted to know if it's pobbile customize the storefront from admin panel shopware, after it will be done.
Thanks.

**Comment by BrocksiNet** [comment]
Hi @vitalijalbu,

we have a vercel commerce template for React that you can have a look at [here](https://github.com/shopwareLabs/vercel-commerce).

It is also possible to use the CMS functions of Shopware Core to change the frontend content. But this function is very simple to integrate into the React template. More CMS features are complete in the Vue demo store.

Best,
Björn.
